pkgs/desktops/gnome/extensions/extensionOverrides.nix

```
@@ -166,4 +167,11 @@ super: lib.trivial.pipe super [
         --replace "GLib.build_filenamev([GLib.DIR_SEPARATOR_S, 'usr', 'share', 'touchegg', 'touchegg.conf'])" "'${touchegg}/share/touchegg/touchegg.conf'"
     '';
   }))
+
+  (patchExtension "custom-command-list@storageb.github.com" (old: {
+    postPatch = ''
+        substituteInPlace "extension.js" \
+          --replace-fail "\"/bin/bash\"" "${bash}/bin/bash"
```

Don't forget the quotes in replacement:

```suggestion
          --replace-fail '"/bin/bash"' '"${bash}/bin/bash"'
```